sgxs-tools/src/sgx_detect/main.rs

```diff
@@ -30,42 +30,40 @@
 //! supported may print debug information to guide the user on how to fix that
 //! item. The debugging may occur interactively.
 
-#[macro_use]
-extern crate log;
```

Comment:
`#[macro_use] extern crate` is totally acceptable even in 2018 edition, especially for things that are imported crate-wide like logging macros.